akka-http-core/src/test/java/akka/http/javadsl/model/EntityDrainingTestCases.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model;
+
+import akka.Done;
+import akka.actor.ActorSystem;
+import akka.japi.function.Procedure;
+import akka.stream.ActorMaterializer;
+import akka.stream.javadsl.Sink;
+import akka.stream.javadsl.Source;
+import akka.util.ByteString;
+import org.junit.Test;
+import org.scalatest.junit.JUnitSuite;
+import scala.util.Try;
+
+import java.util.Arrays;
+import java.util.concurrent.CompletableFuture;
+
+import static org.junit.Assert.assertEquals;
+
+public class EntityDrainingTestCases extends JUnitSuite {
+
+    private ActorSystem sys = ActorSystem.create("test");
```

Comment:
Thinking aloud - could the systems clash if given the same names? Maybe it would be more readable to name it after the test file?